### ISSUE TITLE
Are optional scopes supported?

### ISSUE THREAD

**Issue by emptyflask** [exop-group/doorkeeper-device_authorization_grant#3]
@marco-nicola sorry to bug you, but it seems like optional scopes are ignored -- when specifying scopes on the `authorize_device` request, e.g. `client_id=xxxxxx&scope=write`, the returned access token only includes the default scope:

```json
{
  "access_token": "xGHWsOTDfnbkAlMpZ2im5RwrUm2kYbFfZiyj-2wlX5Y",
  "token_type": "Bearer",
  "expires_in": 7200,
  "scope": "public",
  "created_at": 1610392346
}
```

**Comment by marco-nicola** [comment]
Hi, don't worry, on the contrary, thanks for opening this issue.

Scopes were supposed to be supported, but there's surely a bug!

I should be already on the right track for fixing it... stay tuned!

**Comment by marco-nicola** [comment]
Hey, that was a flagrant mistake indeed.

It should be fixed now, in version `v0.2.1`. It would be really cool if you can give it a try and report here whether it works, or maybe there is still something I missed.

Thanks!

**Comment by emptyflask** [comment]
Thanks! That fixed it for me.
